### DISCUSSION TITLE
Amiga Assembly version 1.7.0 (pre-release) - 1.8.0 (stable)

### DISCUSSION THREAD

**Discussion by prb28** [prb28/vscode-amiga-assembly#231] [General]
New feature by @grahambates : https://github.com/prb28/vscode-amiga-assembly/wiki/BLTCON-Helper

**Comment by phaze101** [comment]
Hi, what one needs to do to view the BLTCON-Helper in Visual Code?

**Comment by Real-MJoe** [comment]
Simple searching "BLT" at command:
![grafik](https://user-images.githubusercontent.com/40276633/183436993-e190343e-24e8-41b1-9cc0-b31abe782cdc.png)

**Reply by phaze101** [reply]
Thanks

**Comment by prb28** [comment]
I'm sorry, I just sent the 1.8.0 as stable too soon (missed the -pre-realease option while publishing). I hope it will work well for every one.
I'll try to be as fast as I can to fix new bugs.
If it's a mess you may still go back to an older version.
